docs/tutorials/ruby-driver-crud-operations.txt

```diff
@@ -460,3 +460,14 @@ Deleting
 
   result = artists.delete_many(:label => 'Mute')
   result.deleted_count # Returns the number deleted.
+
+A Note about the BSON Symbol type
+---------------------------------
+
+Because BSON specification deprecated the BSON symbol type, the `bson` gem will
```

Comment:
Should this say "Because *the* BSON specification"?

Reply:
Fixed